--- a/src/templates.js
+++ b/src/templates.js
@@ -5,7 +5,7 @@ export const mainTemplate = `{{> header}}
 ### ⚠ {{title}}

 {{#each notes}}
-* {{#if commit.scope}}**{{commit.scope}}:** {{/if}}{{text}}
+- {{#if commit.scope}}**{{commit.scope}}:** {{/if}}{{text}}
 {{/each}}
 {{/each}}
 {{/if}}
@@ -23,9 +23,9 @@ export const mainTemplate = `{{> header}}
 `

 export const headerPartial = `## {{#if @root.linkCompare~}}
-  [{{version}}]({{compareUrlFormat}})
+  [{{currentTag}}]({{compareUrlFormat}})
 {{~else}}
-  {{~version}}
+  {{~currentTag}}
 {{~/if}}
 {{~#if title}} "{{title}}"
 {{~/if}}
@@ -33,7 +33,7 @@ export const headerPartial = `## {{#if @root.linkCompare~}}
 {{/if}}
 `

-export const commitPartial = `*{{#if scope}} **{{scope}}:**
+export const commitPartial = `-{{#if scope}} **{{scope}}:**
 {{~/if}} {{#if subject}}
   {{~subject}}
 {{~else}}
