diff --git a/src/templates.js b/src/templates.js
index 577f8be005dbf885289b19788c493ffc775e5394..bafe6b93d65031c9b13bdeef3132c50b46ddeed5 100644
--- a/src/templates.js
+++ b/src/templates.js
@@ -23,9 +23,9 @@ export const mainTemplate = `{{> header}}
 `
 
 export const headerPartial = `## {{#if @root.linkCompare~}}
-  [{{version}}]({{compareUrlFormat}})
+  [{{currentTag}}]({{compareUrlFormat}})
 {{~else}}
-  {{~version}}
+  {{~currentTag}}
 {{~/if}}
 {{~#if title}} "{{title}}"
 {{~/if}}
@@ -33,7 +33,7 @@ export const headerPartial = `## {{#if @root.linkCompare~}}
 {{/if}}
 `
 
-export const commitPartial = `*{{#if scope}} **{{scope}}:**
+export const commitPartial = `-{{#if scope}} **{{scope}}:**
 {{~/if}} {{#if subject}}
   {{~subject}}
 {{~else}}
