--- a/dist/execution-engine/node-execution-context/utils/request-helpers/authentication.js
+++ b/dist/execution-engine/node-execution-context/utils/request-helpers/authentication.js
@@ -45,7 +45,7 @@ async function httpRequestWithAuthentication(credentialsType, requestOptions, wo
         return await di_1.Container.get(backend_network_1.OutboundHttp).requests().request(requestOptions);
     }
     catch (error) {
-        if (error.response?.status === 401 &&
+        if ((error.response?.status === 401 || error.response?.status === 403) &&
             additionalData.credentialsHelper.preAuthentication !== undefined &&
             credentialsDecrypted !== undefined) {
             try {
