diff --git a/dist/execution-engine/node-execution-context/utils/request-helper-functions.js b/dist/execution-engine/node-execution-context/utils/request-helper-functions.js
index 628fce974a7a2fe9ca2a357d89b0b4dae307a919..56fe2798062c6beceb8cc11a9a2df0ae43807f38 100644
--- a/dist/execution-engine/node-execution-context/utils/request-helper-functions.js
+++ b/dist/execution-engine/node-execution-context/utils/request-helper-functions.js
@@ -799,7 +799,7 @@ async function httpRequestWithAuthentication(credentialsType, requestOptions, wo
         return await httpRequest(requestOptions, additionalData.ssrfBridge);
     }
     catch (error) {
-        if (error.response?.status === 401 &&
+        if ((error.response?.status === 401 || error.response?.status === 403) &&
             additionalData.credentialsHelper.preAuthentication !== undefined) {
             try {
                 if (credentialsDecrypted !== undefined) {
