diff --git a/dist/load-nodes-and-credentials.js b/dist/load-nodes-and-credentials.js
index 7c757059ee24bf39a54ecf3113332399ddf1fd8f..e0e1be6ae6bf668d159fde6b4516ef78d78e257a 100644
--- a/dist/load-nodes-and-credentials.js
+++ b/dist/load-nodes-and-credentials.js
@@ -476,12 +476,23 @@ let LoadNodesAndCredentials = class LoadNodesAndCredentials {
             const watchPaths = loader.isLazyLoaded ? [path_1.default.join(directory, 'dist')] : [directory];
             const customNodesRoot = path_1.default.join(directory, 'node_modules');
             if (loader.packageName === n8n_core_1.CUSTOM_NODES_PACKAGE_NAME) {
-                const customNodeEntries = await promises_1.default.readdir(customNodesRoot, {
+                const customNodesRootEntries = await promises_1.default.readdir(customNodesRoot, {
                     withFileTypes: true,
                 });
-                const realCustomNodesPaths = await Promise.all(customNodeEntries
+                const customNodesEntries = [];
+                for (const entry of customNodesRootEntries) {
+                    if (entry.isDirectory() && entry.name.startsWith('@')) {
+                        const scopedCustomNodesEntries = await promises_1.default.readdir(path_1.default.join(entry.parentPath, entry.name), {
+                            withFileTypes: true,
+                        });
+                        customNodesEntries.push(...scopedCustomNodesEntries);
+                    } else {
+                        customNodesEntries.push(entry);
+                    }
+                }
+                const realCustomNodesPaths = await Promise.all(customNodesEntries
                     .filter((entry) => (entry.isDirectory() || entry.isSymbolicLink()) && !entry.name.startsWith('.'))
-                    .map(async (entry) => await promises_1.default.realpath(path_1.default.join(customNodesRoot, entry.name)).catch(() => null)));
+                    .map(async (entry) => await promises_1.default.realpath(path_1.default.join(entry.parentPath, entry.name)).catch(() => null)));
                 watchPaths.push.apply(watchPaths, realCustomNodesPaths.filter((path) => !!path));
             }
             this.logger.debug('Watching node folders for hot reload', {
